src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -27,7 +27,7 @@ public class Xthor : BaseCachingWebIndexer
         private static string ApiEndpoint => "https://api.xthor.tk/";
 
         public override string[] LegacySiteLinks { get; protected set; } = new string[] {
-            "https://xthor.bz/",
+            "https://xthor.to/",
         };
```

Comment:
Ah no, leave the old one there.
the purpose of this block is to automatically replace any config's with one of these old-sitelinks with the new sitelink defined in this indexer.
this way users with existing configs do not need to manually edit their configs to update ;-)

Reply:
Oh ok, i understood thanks!
This is fixed, the old domain name is no longer accessible, can you merge?